src/state/redux.ts

```diff
@@ -127,7 +127,7 @@ const reducers = combineReducers<StoreType>({
 const sagasMiddleware = createSagasMiddleware()
 
 const middleware: typeof composeWithDevTools =
-  process.env.NODE_ENV !== 'production' ? composeWithDevTools : (x: any) => x
+  (import.meta as any).env.NODE_ENV !== 'production' ? composeWithDevTools : (x: any) => x
```

Comment:
Is `NODE_ENV` defined in meta? I think that Vite has a variable to define if it's a production or a development environment